src/main/java/teammates/ui/request/FeedbackSessionUpdateRequest.java

```diff
@@ -1,8 +1,44 @@
 package teammates.ui.request;
 
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.Map;
+
 /**
  * The update request of a feedback session.
  */
 public class FeedbackSessionUpdateRequest extends FeedbackSessionBasicRequest {
 
+    private Map<String, Long> studentDeadlines;
+    private Map<String, Long> instructorDeadlines;
```

Comment:
I think we also need a `notifyUsers` boolean field. I believe instructors have an option of not notifying students of the extension if necessary (at least I know there is a checkbox for this in the mockup).